source/multicast/multicast.rst

```diff
@@ -16,14 +16,17 @@ Creation in OS2IoT
 -------------------
 If you want to make a multicast-group in OS2IoT you have to go into a specific application and then create the multicast-group from there.
 
-You have to fill out all the required forms, and in the end you can add some devices. It's required that the devices share the same service profile, which is set when you create a LoRaWAN device.
+You have to fill out all the required forms, and in the end you can add some devices. It's required that the devices share the same application in chirpstack.
+Before the Chirpstack update to Chirpstack V4, it was based on Service Profiles, but since Service Profiles are not part of Chirpstack V4, it is now based on the application under which your devices are created.
```

Comment:
Again I don't think descriptions of the changes in V4 should be here, but in a seperate section with all the changes

Reply:
That way it will also be easier for people who read the docs for the migration to find everything relevant in the same place

Reply:
I agree with you. But in this specific case, I think it's important to make clear, that an error might occur when creating multicast groups with devices, if the devices was created before the upgrade to V4. The section is called "Creation in OS2IoT".

I could cut down the sentence, but i'm worried that it could be hard to understand correctly (the flow with creating was pretty weird before". Let's discuss it later.